llvm/lib/Target/AMDGPU/SIInstrInfo.cpp

```diff
@@ -4301,7 +4311,9 @@ static bool shouldReadExec(const MachineInstr &MI) {
   if (SIInstrInfo::isVALU(MI)) {
     switch (MI.getOpcode()) {
     case AMDGPU::V_READLANE_B32:
+    case AMDGPU::SI_RELOAD_S32_FROM_VGPR:
```

Comment:
Not sure we want this?

Reply:
To avoid MIR verifier error. Even though readlane/writelane are VALU operations, we don't add implicit EXEC operand to them.